tasks/Task.cpp

```diff
@@ -95,6 +96,16 @@ void Task::writeSpeedCommand(float cmd)
         m_cmd_deadline = base::Time::now() + m_cmd_timeout;
     }
 }
+bool Task::checkSpeedSaturation(base::commands::Joints const& cmd)
+{
+    if (!m_limits.elements.empty()) {
```

Comment:
Avoid this if by initializing `m_limits` with infinity if the property is not set, in the configureHook:

```
if (m_limits.elements.empty()) {
   // Initialize limits with +/-infinity
}
else {
   // write the command to the driver
}
```